.github/workflows/main.yml

```diff
@@ -59,10 +59,14 @@ jobs:
           just docker-build prod
           just docker-build dev
 
+      - name: Check migrations
+        run: |
+          just docker-check-migrations
```

Comment:
note: This also runs the migrations, so, in principle, the `--migrations` below is not strictly necessary, but maybe better to keep it in case this line ever gets removed.

Reply:
https://github.com/opensafely-core/opencodelists/blob/b167ac128618d388ec2b20dd0ac49da26d21b70c/.github/workflows/main.yml#L66-L69 this one? Just checking. It seems to make sense to me